content/_guides/clients.md

```diff
@@ -16,57 +14,51 @@ refer to our [connection guide](/guides/connect).
-"graphical clients", "terminal clients" and "bouncers".
+There are dozens of IRC clients to choose from, here are a selection of
+clients that are easy to install and use:
 
-### Graphical clients
+### GUI clients
 
 These are clients that you navigate primarily with your mouse cursor or touch.
-They can be installed on your device directly or accessed in a web browser.
 
-Some commonly chosen graphical clients for personal computers are
-[Hexchat](https://hexchat.github.io/) (Windows, Linux),
-[Konversation](https://konversation.kde.org/) (Windows, Linux, Mac OS) and
-[Textual](https://www.codeux.com/textual/) (MacOS).
+Desktop clients:
 
-On Android phones & tablets,
-[RevolutionIRC](https://github.com/MCMrARM/revolution-irc) is a popular client
-available from [F-Droid](https://f-droid.org/packages/io.mrarm.irc/) and
-[Google Play](https://play.google.com/store/apps/details?id=io.mrarm.irc).
+- [Hexchat](https://hexchat.github.io/) - Windows, Linux (free)
+- [LimeChat](http://limechat.net/mac/) - macOS (free)
+- [Konversation](https://konversation.kde.org/) - Windows, Linux (free)
+- [mIRC](http://www.mirc.com/) - Windows (paid)
+- [Textual](https://www.codeux.com/textual/) - macOS (paid)
 
-On Apple iOS devices LimeChat is a popular client available from the
-[App Store](https://apps.apple.com/app/limechat-irc-client/id298766460).
+Mobile clients:
+
+- [LimeChat](https://apps.apple.com/app/limechat-irc-client/id298766460) -
+iOS (paid)
+- [Palaver](https://apps.apple.com/us/app/palaver/id538073623) - iOS (paid)
+- [RevolutionIRC](https://play.google.com/store/apps/details?id=io.mrarm.irc)
+\- Android, also on [F-Droid](https://f-droid.org/packages/io.mrarm.irc/) (free)
 
 ### Terminal clients
 
-Terminal or text based clients are clients which you primarily navigate with
-keyboard shortcuts. As these typically do not need mouse navigation, they are
+Terminal or text-based clients are clients which you primarily navigate with
+keyboard shortcuts. These typically do not need mouse navigation, so they are
 great for running on servers and accessing remotely.
 
-[WeeChat](https://weechat.org/) is a popular terminal client which has an
-experimental mouse mode option which makes some operations possible with your
-mouse.
-
-[Irssi](https://irssi.org/) is another popular and mature terminal client.
-
-### Bouncer clients
-
-Bouncers are clients which act as a hub between other clients and the IRC
-network. They allow you to see one connection from multiple places
-(i.e. phone, work computer, home computer).
-
-[ZNC](https://znc.in) is a popular stand-alone bouncer.
+All of these are free and run under Linux, macOS, and Windows (using WSL or Cygwin)
 
-Some other clients have bouncers built in to them but it is not their main
-purpose. These include [Quassel](https://quassel-irc.org/),
-[WeeChat](https://weechat.org/), and [Irssi](https://irssi.org/).
+- [glirc](https://hackage.haskell.org/package/glirc)
```

Comment:
My client is probably too niche to merit an appearance on the page targeted at first-time users. It attempts to fully expose the gory details of IRC. There are certainly some ease-of-use features like live command recognition with parameter placeholders, but I suspect that doesn't cut it.

Reply:
Fair enough, I removed it